stable/mlrun/Chart.yaml

```diff
@@ -1,6 +1,7 @@
 apiVersion: v1
 name: mlrun
-version: 0.2.0
+version: 0.3.0
+appVersion: 0.4.3
```

Comment:
I can see the convenience of having the `appVersion` and the `version` next to each other, but if you're aligning it to be that way, why for this specific chart only ? why not for all charts ?

Reply:
This is meaningless, did this for aesthetics, it's weird to see the `appVersion` at the end. Not gonna enforce a specific order of all fields for all charts though since it's not well defined and... meaningless really